pages/collective-page.js

```diff
@@ -160,6 +161,11 @@ class CollectivePage extends React.Component {
       return <Custom404 />;
     }
 
+    const showCrowdfundingPreviewBanner =
+      !['ROOT', 'ORGANIZATION', 'FUND', 'INDIVIDUAL', 'USER'].includes(collective?.type) &&
```

Comment:
Is `ROOT` a real type that can happen here? I see some hacks forcing it in the frontend dashboard, but nothing in constant files.

Reply:
Nope this can't happen, good catch! Brought over from the dashboard page, fixing this.